lms/djangoapps/certificates/migrations/0011_certificatetemplate_alter_unique.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+from django.db import migrations, models
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+log = logging.getLogger(__name__)
+
+def revert_alter_unique(apps, schema_editor):
+    CertificateTemplateModel = apps.get_model("certificates", "CertificateTemplate")
+
+    # get all unique sets of (org_id, course_key, and mode)
+    all_unique_templates_ignoring_language = CertificateTemplateModel.objects.values_list(
+        "organization_id",
+        "course_key",
+        "mode").distinct()
+
+    # for each course specific template by org_id, course_key and mode
+    for org_id, course_key, mode in all_unique_templates_ignoring_language:
+        # if org_id and course_key and mode:
+        key = CourseKey.from_string(course_key) if course_key else CourseKeyField.Empty
+        course_templates = CertificateTemplateModel.objects.filter(organization_id=org_id, course_key=key, mode=mode)
+        if course_templates.count() > 1:  # theres more than one template for a course
+            # remove all templates past the first (null or default languages are ordered first)
+            language_specific_course_templates = course_templates.order_by('language')[1:] 
+            language_specific_course_template_ids = language_specific_course_templates.values_list('id', flat=True)
+            for template in language_specific_course_templates:
+                log.info('Deleting template ' + str(template.id) +  ' with language ' + str(template.language))
```

Comment:
Can we log out the entire record?  Name, template contents, etc.  That way if someone needs to reconstruct things after it is a little easier